src/artemis/experiment_plans/tests/test_fast_grid_scan_plan.py

```diff
@@ -366,3 +368,31 @@ def test_when_exception_occurs_during_running_then_eiger_disarmed(
         )
 
     fake_fgs_composite.eiger.disarm_detector.assert_called_once()
+
+
+# Eiger is armed if eiger.armed_status is complete and fan ready is called. This test is very slow - could mocking more functions could speed it up
+def test_fgs_arms_eiger_without_grid_detect(
+    fake_fgs_composite: FGSComposite,
+    test_fgs_params: FGSInternalParameters,
+    mock_subscriptions: FGSCallbackCollection,
+    RE: RunEngine,
+):
+    def get_good_status():
+        status = Status()
+        status.set_finished()
+        return status
+
+    fake_fgs_composite.eiger.odin.check_odin_state = MagicMock(return_value=True)
+    fake_fgs_composite.eiger.odin.check_odin_initialised = MagicMock(
+        return_value=[True, True]
+    )
+    fake_fgs_composite.eiger.set_odin_pvs = MagicMock(return_value=get_good_status())
+    fake_fgs_composite.eiger.stale_params.sim_put(0)
+    fake_fgs_composite.eiger._wait_fan_ready = MagicMock(return_value=get_good_status())
+    fake_fgs_composite.eiger._wait_for_odin_status = MagicMock(
+        return_value=get_good_status()
+    )
+
+    RE(bps.stage(fake_fgs_composite.eiger))
+    fake_fgs_composite.eiger.arming_status.wait(60)
+    fake_fgs_composite.eiger._wait_fan_ready.assert_called_once()
```

Comment:
Must: I think the test needs to assert that after the `stage` we're armed without needing to do a wait (to get it consistent we may need one of the mock sets to take some fraction of time)